chia/full_node/mempool_manager.py

```diff
@@ -118,19 +119,25 @@ def compute_assert_height(
 
 
 def run_for_cost_and_additions(
-    coin_id: bytes32, puzzle_reveal: SerializedProgram, solution: SerializedProgram, max_cost: uint64
+    coin_id: bytes32,
+    puzzle_reveal: SerializedProgram,
+    solution: SerializedProgram,
+    max_cost: uint64,
```

Comment:
instead of passing in `max_cost`, I think you could use `npc_result.cost`. Would that work?

Reply:
I checked how mempool items' cost is defined and indeed npc_result's cost can be used here. Done.